.travis.yml

```diff
@@ -90,14 +90,11 @@ stage_osx: &stage_osx
           cd ../../..
           python-build $PYTHON_VERSION ~/python-interpreters/$PYTHON_VERSION
         fi
+        sudo pip2 install -U virtualenv pip setuptools
```

Comment:
sudo and pip2? Isn't pip3 already installed?

Reply:
it's a weird conflict, system python is still python2 and we still don't really have an env setup yet, just a built version of python3. To use virtualenv on python2 system python to build a venv for the pyenv built python3 version. It's kinda hacky (but it's works) and what we were already using. I tried it without this line in https://github.com/Qiskit/qiskit-aer/pull/659/commits/5e5a6286766c792467f685337a2a4561c2c07a02 and the job failed because virtualenv was not installed for the system python.